code/modules/clothing/under/accessories/accessory.dm

```diff
@@ -7,7 +7,12 @@
 	overlay_state = null
 	slot_flags = SLOT_TIE
 	w_class = ITEMSIZE_SMALL
+
 	var/slot = ACCESSORY_SLOT_GENERIC
+
+	/// When attached to another piece of clothing, it'll increase the size by this amount. the final result is rounded up
```

Comment:
Specify the defines it accepts (ACCESSORY_WEIGHT_*), and where to find them (code/__DEFINES/accessories.dm)